src/components/paradox/results/personality/career/details/style.scss

```diff
@@ -0,0 +1,270 @@
+@import "~style/paradox/helpers";
+
+.container {
+  @extend %widgetContainer;
+  @extend %cf;
+  @extend %container;
+
+  display: block;
+  margin: 0 auto 5%;
+  padding: 0;
+  width: 262px;
+  z-index: 1;
+  
+  @include min-width(xs) {
+    margin: 0 0% 3%;
+    @include cols(6);
+    width: 47%;
+    padding: 0;
+  }
+  @include min-width(md) {
+    width: 30%;
+  }
+  @include min-width(xl) {
+    width: 23%;
+    margin: 0 1% 1.5%;
+  }
+  img { 
+    margin: 0px 20px 0px 0px;
+    border-radius: 10px;
```

Comment:
We also have various $border-radius variables to match paradox brand styles

Reply:
OK now using $border-radius variables!